Application/Services/TraderManagementService.cs

```diff
@@ -0,0 +1,50 @@
+using Domain.Entities;
+using Domain.Interfaces;
+using Application.Interfaces;
+using Application.Validations;
+using FluentValidation;
```

Comment:
you don't use fluent validation here, check for unnessary usings.
but I want you to use a fluent validation here.